lib/Mojo/Transaction.pm

```diff
@@ -242,8 +252,11 @@ False, this is not a L<Mojo::Transaction::WebSocket> object.
   my $address = $tx->remote_address;
   $tx         = $tx->remote_address('127.0.0.1');
 
-Same as L</"original_remote_address"> or the last value of the C<X-Forwarded-For> header if L</"req"> has been
-performed through a reverse proxy.
+Same as L</"original_remote_address"> unless the L</"req"> has been performed via a
```

Comment:
`unless the L</"req">` sounds weird, `unless L</"req">` would be similar to how it was before.

Reply:
I was reading it as "unless the request" but as code losing the "the" is probably better. I'm fine with either.